packages/agent-toolkit/src/core/tools/platform-api-tools/workforms-tools/workforms.consts.ts

```diff
@@ -59,19 +88,21 @@ export const GraphQLDescriptions = {
     operations: {
       updateFormSettings: 'Update form configuration including features, appearance, and accessibility options.',
       setFormPassword:
-        'Set a password on a form to restrict access. This will enable password protection for the form.',
+        'Set a password on a form to restrict access. This will enable password protection for the form. Required for the action "setFormPassword" in the update form tool.',
       shortenUrl: 'Shorten a URL for a form and store it in the form settings. Returns the shortened link object.',
     },
     properties: {
       features:
-        'Object containing form features including but not limited to password protection, response limits, login requirements, etc.',
-      appearance: 'Object containing visual styling including colors, layout, fonts, and branding elements.',
-      accessibility: 'Object containing accessibility options such as language, alt text, etc.',
+        'Object containing form features including but not limited to password protection, response limits, login requirements, etc. Required when updating the features of the form.',
+      appearance:
+        'Object containing visual styling including colors, layout, fonts, and branding elements. Required when updating the appearance of the form.',
+      accessibility:
+        'Object containing accessibility options such as language, alt text, etc. Required when updating the accessibility of the form.',
       isInternal: 'Boolean indicating if the form is restricted to internal users only.',
       reCaptchaChallenge: 'Boolean enabling reCAPTCHA verification to prevent spam submissions.',
       password: 'Object containing password protection configuration for the form.',
       passwordEnabled:
-        'Boolean enabling password protection. When true, users must enter a password to access the form.',
+        'Boolean disabling password protection. Can only be updated to false, to enable password protection, use the setFormPassword action instead.',
```

Comment:
```suggestion
        'Boolean disabling password protection. Can only be updated to false. In order to enable password protection use the setFormPassword action instead.',
```

Make sure your descriptions don't have any ambiguity